qa/distros/podman/centos_8.stream_container_tools.yaml

```diff
@@ -0,0 +1,14 @@
+os_type: centos
+os_version: "8.stream"
+overrides:
+  selinux:
+    whitelist:
+      - scontext=system_u:system_r:logrotate_t:s0
+
+tasks:
+- pexec:
+    all:
+    - sudo cp /etc/containers/registries.conf /etc/containers/registries.conf.backup
+    - sudo dnf -y  module reset container-tools
+    - sudo dnf -y  module install container-tools
```

Comment:
LGTM.
I tried this on my CentOS Stream release 8 system and got podman version 3.4.1-dev. 
In TripleO we're still enabling container-tools:3.0 [1] and getting podman version 3.0.2-dev. 
If this works for you, I think it's good ceph will be ahead. I expect TripleO will catch up.

[1] https://github.com/openstack/tripleo-quickstart/blob/master/config/release/centosci/master-current-tripleo.yml#L69-L70